app/assets/stylesheets/home/components/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$text-color: #0077B6;
+$breakpoint-desktop: 992px;
+
+@mixin desactivated_navbarMobile{
+  .navbar{
+    display: none;
+  }
+  .navbar-expand-lg .navbar-collapse{
+    display: none !important;
+  }
+}
```

Comment:
Empty lines at the end of files ensure compatibility between different code editors and different operating systems. Most modern editors can add empty lines at the end of the line by themselves if they're told to do so.